src/App.tsx

```diff
@@ -15,48 +17,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  name = 'name',
+  length = 'length',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: string,
+  isRev: boolean,
+): string[] {
+  const preparedGoods = [...goods];
+
+  preparedGoods.sort((good1, good2) => {
+    switch (sortField) {
+      case SortType.length:
+        return good1.length - good2.length;
+
+      case SortType.name:
+        return good1.localeCompare(good2);
+
+      default:
+        return 0;
+    }
+  });
+
+  if (isRev) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
+  const [isRev, setIsReversed] = useState(false);
```

Comment:
Don't cut variable naming 
```suggestion
  const [isReverse, setIsReversed] = useState(false);
```